src/aliceVision/voctree/databaseIO.hpp

```diff
@@ -23,18 +24,18 @@ namespace voctree {
  * @brief Given a vocabulary tree and a set of features it builds a database
  *
  * @param[in] fileFullPath A file containing the path the features to load, it could be a .txt or an AliceVision .json
- * @param[in] descFolder The folder containing the descriptor files (optional)
+ * @param[in] featuresFolders The folder(s) containing the descriptor files (optional)
  * @param[in] tree The vocabulary tree to be used for feature quantization
  * @param[out] db The built database
  * @param[out] documents A map containing for each image the list of associated visual words
  * @param[in] Nmax The maximum number of features loaded in each desc file. For Nmax = 0 (default), all the descriptors are loaded.
  * @return the number of overall features read
  */
 template<class DescriptorT, class VocDescriptorT>
-std::size_t populateDatabase(const sfm::SfMData &sfmData,
-                             const std::string &descFolder,
-                             const VocabularyTree<VocDescriptorT> &tree,
-                             Database &db,
+std::size_t populateDatabase(const sfm::SfMData& sfmData,
+                             const std::vector<std::string>& featuresFolders,
```

Comment:
instead of changing the API for all these functions, why not declare a second function with the std::vector and figure out the way to call each other. 
If one wishes to call the function with just a string he nedds to create a vector and pass to it